config/projects-to-test/openjdk19-excluded.files

```diff
@@ -0,0 +1,589 @@
+<!-- This file was generated by script at https://github.com/checkstyle/checkstyle/issues/11185#issue-1099900699 -->
+<!-- Excluded due to exception; issues opened -->
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8654-->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]jdk[\\/]javadoc[\\/]doclet[\\/]testSupplementary[\\/]C.java$"/>
+</module>
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]hotspot[\\/]jtreg[\\/]runtime[\\/]exceptionMsgs[\\/]methodPrinting[\\/]TestPrintingMethods.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8650 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]MethodParameters[\\/]UncommonParamNames.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8655 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]UnicodeAtEOL.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8656 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]UnicodeCommentDelimiter.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/8654 -->
+<module name="BeforeExecutionExclusionFileFilter">
+  <property name="fileNamePattern" value="[\\/]test[\\/]langtools[\\/]tools[\\/]javac[\\/]unicode[\\/]FirstChar2.java$"/>
+</module>
+
+<!-- until https://github.com/checkstyle/checkstyle/issues/12515 -->
+<!-- until https://github.com/checkstyle/checkstyle/issues/12507 -->
```

Comment:
All pattern related files are here, with some mixed syntax, so we might have to wait until final Java 19 syntax update to remove suppressions.